package.json

```diff
@@ -4,15 +4,19 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "predev": "yarn generate",
     "build": "next build",
+    "prebuild": "yarn generate",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "generate": "graphql-codegen"
```

Comment:
Adding the scripts to generate the types, and ensure that we always generate them before dev/build